src/components/cookie-banner/index.md

```diff
@@ -200,6 +182,21 @@ You can use this example text for a service that set:
 
 You’ll need a [Cookies page in your service](/patterns/cookies-page/) as well as a cookie banner.
 
+## Brand refresh of the Cookie banner component
+
+In June 2025, we updated this component to support a wider refresh of the GOV.UK brand.
+
+The updated Cookie banner component uses light blue as the background colour, instead of grey.
+
+To help teams refresh the GOV.UK brand in their services, we released GOV.UK Frontend v5.10.0 (and later fix versions). For teams on earlier versions, we released GOV.UK Frontend v4.10.0
+
+To see more details and how to update, you can read:
+
+- [release notes for v5.10.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.10.0)
```

Comment:
I don't think we need to include the fix versions here (otherwise we're going to have to keep updating it).

I think we just do 5.10 and 4.10